tools/profiles/debug.json

```diff
@@ -42,7 +42,7 @@
             "--diag_suppress=Pa050,Pa084,Pa093,Pa082", "-On", "-r", "-DMBED_DEBUG",
             "-DMBED_TRAP_ERRORS_ENABLED=1"],
         "asm": [],
-        "c": ["--vla"],
```

Comment:
Yeah, you are correct. I tried using them in common location at *.json files and got errors. Will verify once.